apps/meteor/ee/app/livechat-enterprise/server/lib/Helper.js

```diff
@@ -301,7 +259,11 @@ export const getLivechatQueueInfo = async (room) => {
 		return null;
 	}
 
-	const [inq] = await LivechatInquiry.getCurrentSortedQueueAsync({ _id, department });
+	const [inq] = await LivechatInquiry.getCurrentSortedQueueAsync({
+		_id,
```

Comment:
looks like `getCurrentSortedQueueAsync` doesn't accept an `_id` param 🤔  do you mean `inquiryId` instead?